snyk/managers.py

```diff
@@ -178,17 +178,14 @@ def all(self, page: int = 1):
             "total"
         ]  # contains the total number of results (for pagination use)
 
-        results = dependency_data["results"]
+        results = [self.klass.from_dict(item) for item in dependency_data["results"]]
 
         if total > (page * results_per_page):
             next_results = self.all(page + 1)
             results.extend(next_results)
             return results
```

Comment:
Looks like this return can be removed thanks to refactoring the results var?

Reply:
Thanks. Removed